staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -25,9 +26,28 @@ import (
 	"github.com/google/cel-go/common/types"
 	"github.com/google/cel-go/common/types/ref"
 	"github.com/google/cel-go/common/types/traits"
+
 	"k8s.io/apiserver/pkg/cel"
 )
 
+// panicOnUnknown makes cost estimate functions panic on unrecognized functions.
+// This is only set to true for unit tests.
+var panicOnUnknown = false
```

Comment:
@jpbetz I got bit when implementing cost tests on the selector functions and was hitting unknown paths and the cost estimates were silently defaulting to 0 / 1, etc... I added this to make those unhandled paths loud in unit test mode ... thoughts on this?

Reply:
Something like this is needed.  Turning this on for all our existing cost tests would be great.

Reply:
It might be possible to come up with a test that ensures we try out every library function with this enabled. I'd be willing to take that on as a follow up.